tdrs-backend/Pipfile

```diff
@@ -39,7 +39,6 @@ django-csp = "==3.7"
 djangorestframework = "==3.13.1"
 drf-yasg = "==1.20.0"
 gunicorn = "==20.1.0"
-ipdb = "==0.13.9"
```

Comment:
Does use of `shell_plus` in cloud.gov environments utilize this?

Reply:
deployed to RAFT and started shell_plus with no problem:

![Screenshot 2024-09-03 at 11 55 17 AM](https://github.com/user-attachments/assets/8e5ac4fd-20c6-4e08-87a9-e4f4252c8c34)

Reply:
`ipdb` isn't imported anywhere and i can't tell if any of our other packages use it. the tool itself seems to be some sort of debugger - https://pypi.org/project/ipdb/

all the tests pass including the ones with `log`. along with mo's successful deployment, that mitigates my immediate concerns. i'd be curious to know what we use this for, if anything. i'm suspicious it may have just been part of the 'default' packages first installed for the project if it's no longer relevant

Reply:
it is a replacement for pdb, but I have never used it. I always use pdb.

Reply:
also.. i've just seen this reproduced in our a11y environment. tests passed in circleci prior to the deployment being initiated. is this failure limited to cloud.gov for some reason?

Reply:
yes, limited to cloud.gov. Looks like the version of linux in cloud.gov has a conflict with ipdb version and doesn't allow the build.

Reply:
Also note I could bump the version (and maybe resolve this issue), but decided to remove a package that is not used.